GamebookEngine/Views/Introduction/IntroductionView.swift

```diff
@@ -131,7 +131,7 @@ struct IntroductionView: View {
             }
         }
         .background(Color(UIColor(named: "background")!))
-        .ignoresSafeArea(.all, edges: .vertical)
```

Comment:
Didn't look into whether this causes issues or not... need to take a peek.

Reply:
![image](https://user-images.githubusercontent.com/1126616/231655915-310fd9ab-a7d8-49f5-af24-497d7909c136.png)
 doesn't seem to cause issues with the background color on an actual device, but looks wonky without it in the canvas preview, weird